src/components/results/generic/style.scss

```diff
@@ -0,0 +1,198 @@
+@import "style/helpers";
+
+.container {
+  @extend %container;
+  color: $-black;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  .header {
+    display: flex;
+    padding: $buffer $buffer-lg;
+    background-color: $-white;
+    border-bottom: 1px solid #DADCE0;
+    justify-content: space-between;
+    .profileDetails {
+      display: flex;
+      flex-direction: row;
+      gap: $buffer;
+      color: #A9A9A9;
+      .profileCircle {
```

Comment:
To allow it to be easier to override if a client wants to, it's better not to nest the classes so deep. I think most of them can be top-level unless otherwise necessary.

Also alphabetize when order not otherwise relevant!